### PR TITLE
todo app

### PR DESCRIPTION
**Pull request by donizer**
[DEMO LINK](https://donizer.github.io/react_todo-app/)

### CODE REVIEW

**Comment by donizer** [comment]
again as in this [case](https://github.com/mate-academy/react_dynamic-list-of-goods/pull/1850) i have the same issue.

During `npm i` it fails to find config file 
`ESLint couldn't find the config "@mate-academy/eslint-config-react-typescript" to extend from. Please check that the name of the config is correct.`

File: src/Contexts/Context.tsx
Comment (loralevitska) [inline src/Contexts/Context.tsx:31]: Remove comment. Try it write obvious code that explain all we need without comments) Fix all cases

File: src/hooks/useLocalStorage.tsx
Comment (loralevitska) [inline src/hooks/useLocalStorage.tsx:10]:
```suggestion
    if (!data) {
```

**Review by loralevitska — CHANGES_REQUESTED** [review]
Great job overall! Just couple tiny things needs to be fixed before approval)

- add right condition `1 item left`
image
- remove outline, it breaks design
- according your question, such files should be correct by default in your project. From which part of project did you run your command (maybe it was not from root file). If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

**Comment by donizer** [comment]
All styles was here initially. And outline present in [reference](https://todomvc.com/examples/vanillajs/)

**Comment by donizer** [comment]
image

File: src/App.tsx
Comment (Andriy-Shymkiv) [inline src/App.tsx:21]: you put date in id field? but why?

File: src/App.tsx
Comment (Andriy-Shymkiv) [inline src/App.tsx:14]: please give a better naming for this type

File: src/Contexts/Context.tsx
Comment (Andriy-Shymkiv) [inline src/Contexts/Context.tsx:67]: please remove logs

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
- this arrow should be black when all todos is completed:
image
 
the codebase looks good. left some comments

File: src/App.tsx
Comment (donizer) [reply src/App.tsx:21]: to use as a unique key 
```
Implement TodoApp component with an input field to create new todos on submit (Enter). Each item should have:

    id - unique identifier (+new Date() is good enough)
    title - the text of a todo
    completed - current status (false by default)
```

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**
